src/compute/mod.rs

```diff
@@ -272,6 +272,12 @@ pub fn compute_hidden_layout(tree: &mut (impl LayoutPartialTree + CacheTree), no
     LayoutOutput::HIDDEN
 }
 
+/// A module for unified re-exports of detailed layout info structs, used by low level API
+#[cfg(feature = "detailed_layout_info")]
+pub mod detailed_info {
+    pub use super::grid::DetailedGridInfo;
+}
+
```

Comment:
This is going to fail if the user enables the `flexbox` and `detailed_layout_info` features but not the `grid` feature. Need to feature flag the re-export on the `grid` feature. Could also consider just re-exporting the type directly into `compute` rather than creating `compute::detailed_info`. This would still need proper feature flagging.

Reply:
We should probably add a CI job for the `detailed_layout_info` feature. Perhaps one with just the (std, grid, detailed_layout_info) features enabled, and one with the (std, grid, detailed_layout_info) features enabled?

Reply:
> This is going to fail if the user enables the `flexbox` and `detailed_layout_info` features but not the `grid` feature. Need to feature flag the re-export on the `grid` feature. Could also consider just re-exporting the type directly into `compute` rather than creating `compute::detailed_info`. This would still need proper feature flagging.

Thanks for pointing the feature mistake! 
I thoughts that this would be cleaner than adding a configuration check for `grid` and `detailed_layout_info` for `DetailedGridInfo` (and other layout info later) in `lib.rs`. What do you think?

Reply:
> We should probably add a CI job for the `detailed_layout_info` feature. Perhaps one with just the (std, grid, detailed_layout_info) features enabled, and one with the (std, grid, detailed_layout_info) features enabled?

Are we considering the cases when `std` is not enabled too?

Reply:
> Are we considering the cases when std is not enabled too?

Probably not for the time being. It would be useful, but it's a balancing act to avoid too many CI jobs.